src/components/Integrations/BigQuery/FreeAccount/styles/index.css

```diff
@@ -0,0 +1,29 @@
+.check-icon {
```

Comment:
@cbernat @fcoronelmakingsense @lucianaDilizio (@GusBaamonde está de vacaciones)

Como ven estos estilos? encajan con las prácticas sobre WebApp o conviene dejar un HTML más simple, incluso sin estilos e implementar esto en la UI Library?

Reply:
For css we use styled components, like here https://github.com/FromDoppler/doppler-webapp/blob/master/src/components/Reports/GDPRPermissions/SubscriberGdpr.styles.js we have some css but it's not the way we are headed. It's old code. Could you make this into a styled component please? Or as @andresmoschini perhaps consider implementing this styles in ui library if they may be reutilized in another screen.

Reply:
- I should try to build a similar UI using what we have in UI library.
- Icons usually are on the library. Maybe, this one, it could be added as Cecilia said (by the moment).
- I should also try to replace things that we don't have in library for something similar (as the item icon).
- The line could be replaced with `<hr>`

Reply:
in a meeting with @lucianaDilizio applied changes for use classname compones from doppler ui, in the image of PR you can see the new result, with out css local